app/views/admin/conferences/_todo_list.html.haml

```diff
@@ -23,7 +23,7 @@
       Set up call for papers
   %li{'class'=>"list-group-item #{class_for_todo(conference_progress['venue'])}"}
     %span{'class'=>icon_for_todo(conference_progress['venue'])}
-    - if can? :update, @conference.venue
+    - if can? :update, @conference
```

Comment:
Yes @differentreality do you suggest something like this:
`-if can? :update, @conference.build_venue`
and then reloading the conference?
I tried this approach first, but the problem I faced with this approach is suppose conference already has a venue then 
`-if can? :update, @conference.build_venue` 
this will delete the venue stored in database and create a new temporary venue for this conference.